models/topics-models

```diff
@@ -53,10 +53,23 @@ function fetchCommentByArticleId(articleIndex) {
     });
 }
 
+function addCommentByArticleId(articleId, username, body) {
+  return db
+    .query(
+      `INSERT INTO comments (body, author, article_id) 
+      VALUES ($1, $2, $3) RETURNING *;`,
+      [body, username, articleId]
+    )
+    .then((result) => {
+      return result.rows[0];
+    });
+}
```

Comment:
Lovely model! This is really nice and simple, meaning that it's easy for me to read and understand! :D